### PR TITLE
added link to portal documentation

### LINKED ISSUE #244: Documentation is missing
Hey All,

Thanks for providing this great feature.
As I am relativley new to SSO and SAML, I would love to reach back to some documentation.
Anything I find online is either outdated or "file not found" for my version of Nextcloud 13.

Where would you recommend me to go looking?

I am trying to setup SSO with SAML against an Azure Active Directory (WestEurope) with a Nextcloud 13 installed from tar-package on an Ubuntu with NFS (Azure File Storage) and a mysql-db on localhost.

Looking forward to your responses.

Best,
Enno

### PR DESCRIPTION
**Pull request by wiswedel**
fixes #244 
continues https://github.com/nextcloud/user_saml/pull/315

Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

### CODE REVIEW

**Comment by wiswedel** [comment]
anybody fancy reviewing?

File: appinfo/info.xml [hunk 1]
```diff
@@ -23,6 +23,9 @@ While theoretically any other authentication provider implementing either one of
 	<types>
 		<authentication/>
 	</types>
+	<documentation>
+		<admin>https://portal.nextcloud.com/article/configuring-single-sign-on-10.html</admin>
```
Comment (skjnldsv) [inline appinfo/info.xml]: Is there no unique direct link?
I think we should not use text like that that can potentially change? :thinking:

Comment (wiswedel) [reply appinfo/info.xml]: Not that I'm aware of. What do you suggest instead?

Comment (skjnldsv) [reply appinfo/info.xml]: Nothing if there is no alternative :p

**Pull request merged**